lib/plurimath/math/function/semantics.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require_relative "binary_function"
+
+module Plurimath
+  module Math
+    module Function
+      class Semantics < BinaryFunction
+        def to_mathml_without_math_tag
+          first_value = parameter_one&.to_mathml_without_math_tag
+          second_value = other_tags(parameter_two)
+          Utility.update_nodes(
+            Utility.ox_element("semantics"),
+            second_value.insert(0, first_value),
+          )
+        end
+
+        def to_latex
+          parameter_one&.to_latex
+        end
+
+        def to_asciimath
+          parameter_one&.to_asciimath
+        end
+
+        def to_html
+          parameter_one&.to_html
+        end
+
+        def to_omml_without_math_tag
+          parameter_one&.to_omml_without_math_tag
+        end
+
+        protected
+
+        def other_tags(array)
```

Comment:
This method is not tested by any spec right now. When I removed the contents of this method, all specs still pass.

Reply:
Confirm fixed. Thanks!